packages/client-sdk-nodejs/src/internal/grpc/middlewares-interceptor.ts

```diff
@@ -75,6 +77,17 @@ export function middlewaresInterceptor(
             status: StatusObject,
             next: (status: StatusObject) => void
           ): void {
+            if (status.code === 4) {
+              const logger = loggerFactory.getLogger('grpc-interceptor');
```

Comment:
maybe make this a class variable so we don't have to get a new logger each time we hit this code path